src/listeners/command-handler/CoreMessageParser.ts

```diff
@@ -49,6 +49,7 @@ export class CoreListener extends Listener<typeof Events.PreMessageParsed> {
 	}
 
 	private getMentionPrefix(message: Message): string | null {
+		if (!this.container.client.disableMentionPrefix) return null;
```

Comment:
It defaults to false. `!false` = `true`
So now by default it will be `true` thus disabling mention prefix for everyone.

Remove the `!`